src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -315,7 +260,7 @@ public void doMove(final Map<Territory, ProTerritory> attackMap, final IMoveDele
 
       // Remove negative value territories
       patd.setValue(attackValue);
-      if (attackValue <= 0 || (isDefensive && attackValue <= 8 && data.getMap().getDistance(myCapital, t) <= 3)) {
+      if (attackValue <= 0 || (isDefensive && attackValue <= 8 && data.getMap().getDistance(ProData.myCapital, t) <= 3)) {
```

Comment:
Some more magic numbers, "8", "3". Not sure what they represent, I have the same recommendation here again of starting by labeling them with a variable assignment